### DISCUSSION TITLE
when autorotate the 3d object you see lines

### DISCUSSION THREAD

**Discussion by jairojiroh** [google/model-viewer#2301] [Q&A]
when autorotate the 3d object you see lines
how can I solve that?
attached image

Thank you

https://ibb.co/M6X09qW

**Comment by elalish** [comment]
I don't think anyone can answer that without a link where we can repro it. I'd guess you have some errant geometry in your glTF.

**Reply by elalish** [reply]
Have you tried this file in our editor? You might be able to figure something out from looking at the materials tab. https://modelviewer.dev/editor/
